packages/ui/components/button/Button.tsx

```diff
@@ -176,11 +180,23 @@ export const Button = forwardRef<HTMLAnchorElement | HTMLButtonElement, ButtonPr
     },
     <>
       {StartIcon && (
-        <StartIcon
-          className={classNames("inline-flex", size === "icon" ? "h-4 w-4 " : "mr-2 h-4 w-4 stroke-[1.5px]")}
-        />
+        <>
+          {size === "fab" ? (
+            <>
+              <StartIcon className="mr-2 hidden h-4 w-4 stroke-[1.5px] sm:inline-flex" />
+              <Icon.FiPlus className="inline h-6 w-6 sm:hidden" />
+            </>
+          ) : (
+            <StartIcon
+              className={classNames(
+                "inline-flex",
+                size === "icon" ? "h-4 w-4 " : "mr-2 h-4 w-4 stroke-[1.5px]"
+              )}
+            />
+          )}
+        </>
       )}
-      {props.children}
+      {size === "fab" ? <span className="hidden sm:inline">{props.children}</span> : props.children}
```

Comment:
Making this `sr-only` instead of hidden will ensure that screenreaders can still read this <3

Reply:
but does it get visible for sm:inline then?